test/routes/service-complete.route.test.js

```diff
@@ -722,7 +722,8 @@ const _createSection2RedisMock = (
 const _createSection10RedisMock = (
   isOwnedByApplicant,
   hasOwnerEmail = true,
-  itemType = ItemType.MUSICAL
+  itemType = ItemType.MUSICAL,
+  itemDescription = 'sword'
```

Comment:
Item description is an object not a string, this mock should return a representative object to avoid confusion

Reply:
Done.